packages/flutter/test/material/tabs_test.dart

```diff
@@ -2350,4 +2350,72 @@ void main() {
     expect(find.text('Tab1'), findsOneWidget);
     expect(find.text('Tab2'), findsOneWidget);
   });
+  testWidgets('DefaultTabController should allow for a length of zero', (
+      WidgetTester tester) async {
+    // Test for https://github.com/flutter/flutter/issues/37139
+    final List<Widget> tabList = <Widget>[];
```

Comment:
I suggested this at the beginning, but realized we could do better here. Instead of using two separate lists of widgets, we could use a List<String> to define both the TabBar and TabView contents instead:
```suggestion
  List<String> tabTextContent = <String>[];
```